docs/kubernetes.aad.md

```diff
@@ -0,0 +1,100 @@
+# Microsoft Azure Container Service Engine - Kubernetes AAD integration Walkthrough
```

Comment:
Docs directory structure has changed. I suggest this is moved to docs/kubernetes/aad.md or features.md.